### DISCUSSION TITLE
Blank page

### DISCUSSION THREAD

**Discussion by Emman-code** [lencx/ChatGPT#521] [Q&A]
I downloaded the software....and after installation when I opened it, only a blank white page was present. But the control menu works.  Without the main page, what is the use of the control page?
![Capture](https://user-images.githubusercontent.com/73483278/221615399-c53e9c0b-753f-4374-8151-a8fe6be7e701.PNG)

**Comment by lencx** [comment]
The application is just a wrapper for openai, this issue is obviously a problem with the openai service, please try again later.
